include/zephyr/bluetooth/conn.h

```diff
@@ -2688,6 +2688,28 @@ struct bt_conn *bt_conn_lookup_addr_br(const bt_addr_t *peer);
  */
 const bt_addr_t *bt_conn_get_dst_br(const struct bt_conn *conn);
 
+/** @brief Change the role of the conn.
+ *
+ *  @param conn Connection object.
+ *  @param role The role that want to switch as.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ *  @return -ENOBUFS HCI command buffer is not available.
+ *  @return -EINVAL @p conn is not a valid @ref BT_CONN_TYPE_BR connection
+ */
+int bt_conn_br_switch_role(const struct bt_conn *conn, uint8_t role);
+
+/** @brief Enable/disable role switch of the connection by setting the connection's link policy.
+ *
+ *  @param conn Connection object.
+ *  @param enable Value allowing/disallowing controller to be role switchable.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ *  @return -ENOBUFS HCI command buffer is not available.
+ *  @return -EINVAL @p conn is not a valid @ref BT_CONN_TYPE_BR connection.
+ */
+int bt_conn_br_set_role_switchable(const struct bt_conn *conn, bool enable);
```

Comment:
Is "switchable" a term that's found in the core spec? If not, I'd avoid introducing it in the API. Perhaps `bt_conn_br_role_switch_allowed()`?

Reply:
It is not from spec.
Is it better to use `bt_conn_br_set_role_switch_enable(const struct bt_conn *conn, bool enable)` like `bt_conn_le_set_tx_power_report_enable` and `bt_conn_le_set_path_loss_mon_enable`?